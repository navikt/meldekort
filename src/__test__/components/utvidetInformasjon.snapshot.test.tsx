import * as React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import ProviderWrapper from '../testSetup/providerWrapper';
import UtvidetInformasjon from '../../app/components/utvidetinformasjon/utvidetInformasjon';

Enzyme.configure({ adapter: new Adapter() });

it('UtvidetInformasjon', () => {
  const tree = shallow(
    <ProviderWrapper>
      <UtvidetInformasjon>
        <span>Test</span>
      </UtvidetInformasjon>
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
