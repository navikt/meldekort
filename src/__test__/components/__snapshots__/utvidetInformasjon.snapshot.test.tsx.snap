// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UtvidetInformasjon 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Connect(IntlProvider)
    defaultLocale="nb"
    locale="nb"
  >
    <UtvidetInformasjon>
      <span>
        Test
      </span>
    </UtvidetInformasjon>
  </Connect(IntlProvider)>
</Provider>
`;
